modules/local/bedtools_genomecov.nf

```diff
@@ -0,0 +1,34 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+//params.options = [:]
+//def options    = initOptions(params.options)
+
+process BEDTOOLS_GENOMECOV {
```

Comment:
Fine as local module for now, but there is now an 'official' nf-core/modules for [this](https://github.com/nf-core/modules/blob/master/modules/bedtools/genomecov/main.nf) to consider for the future (same ofr a few of the other modules here)

Reply:
I will use the bedtools genomecov modules from nf-core/modules in a later release